configs/frontend_config/config.json

```diff
@@ -1 +1,926 @@
+                  }
+                ]
+              },
+              {
+                "tabName": "Service Delivery Model",
+                "headerTitle": "Service Delivery Model",
+                "displayText": "Service Delivery Model",
+                "encounterType": "62ee5922-a229-48d3-a1da-875c1ffa9436",
+                "columns": [
+                  {
+                    "id": "vlDate",
+                    "isDate": true,
+                    "title": "Date",
+                    "concept": "160753AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+                  },
+                  {
+                    "id": "dsdStatus",
+                    "title": "Status",
+                    "concept": "8742967d-8107-4cbb-a17e-9a8c7f624673"
+                  },
+                  {
+                    "id": "dsdModel",
+                    "title": "DSD Model",
+                    "concept": "52824cbe-0e4d-4c18-8179-80b5799f34ed"
+                  },
+                  {
+                    "id": "actions",
+                    "title": "Actions",
+                    "actionOptions": [
+                      {
+                        "formName": "Service Delivery Model Form",
+                        "package": "hiv",
+                        "label": "View Details",
+                        "mode": "view"
+                      },
+                      {
+                        "formName": "Service Delivery Model Form",
+                        "package": "hiv",
+                        "label": "Edit Forms",
+                        "mode": "edit"
+                      }
+                    ]
+                  }
+                ],
+                "launchOptions": {
+                  "displayText": "Add",
+                  "moduleName": ""
+                },
+                "formList": [
+                  {
+                    "name": "Service Delivery Model Form",
+                    "uuid": "1e14f841-b42b-3273-93db-807927ca9a82"
+                  }
+                ]
+              },
+              {
+                "tabName": "Transfer Out",
+                "headerTitle": "Transfer Out",
+                "displayText": "Transfer Out",
+                "encounterType": "transferOutEncounterType_UUID",
+                "columns": [
+                  {
+                    "id": "visitDate",
+                    "isDate": true,
+                    "title": "Visit Date",
+                    "concept": "163137AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+                  },
+                  {
+                    "id": "reasonsForTesting",
+                    "title": "Receiving Facility",
+                    "concept": "receivingFacility_UUID"
+                  },
+                  {
+                    "id": "tranferOutDate",
+                    "title": "Transfer-Out Date",
+                    "isDate": true,
+                    "concept": "TransferOutDate_UUID"
```

Comment:
Is this a usable concept?

Reply:
I forgot to pass the actual uuid, initially we were using concept keys in the UI workflows

Reply:
So, I'd rethink only supporting concept UUIDs. We have an idea of Concept Maps that can be used to give meaningful names to specific concepts, e.g., a concept could just be mapped to `HIV:Transfer Out Date` and that gives the same flexibility I think the concept keys has, with the advantage of only having to add that mapping once.